vita3k/app/src/app_init.cpp

```diff
@@ -111,8 +111,8 @@ bool init(HostState &state, Config cfg, const Root &root_paths) {
     if (state.cfg.pref_path == root_paths.get_pref_path_string() || state.cfg.pref_path.empty())
         state.pref_path = root_paths.get_pref_path_string();
     else {
-        if (state.cfg.pref_path.back() != '/')
-            state.cfg.pref_path += '/';
+        if (state.cfg.pref_path.back() != '\\')
+            state.cfg.pref_path += '\\';
```

Comment:
```
        if (state.cfg.pref_path.back() != '\\' || state.cfg.pref_path.back() != '/')
            state.cfg.pref_path += '/';
```
`/` in path will work on all platforms afaik.
Path can have either missing `\\` or `/`.

Reply:
I see